src/Services/PatientService.php

```diff
@@ -268,27 +251,37 @@ public function getAll($search)
         return $results;
     }
 
+    /**
+     * Returns a single patient record by patient id.
+     */
     public function getOne()
     {
-        $sql = "SELECT id,
-                   pid,
-                   pubpid,
-                   title,
-                   fname,
-                   mname,
-                   lname,
-                   ss,
-                   street,
-                   postal_code,
-                   city,
-                   state,
-                   country_code,
-                   phone_contact,
-                   email,
-                   DOB,
-                   sex,
-                   race,
-                   ethnicity
+        $sql = "SELECT  id,
+                        pid,
+                        pubpid,
+                        title,
+                        fname,
+                        mname,
+                        lname,
+                        ss,
+                        street,
+                        postal_code,
+                        city,
+                        state,
+                        county,
+                        country_code,
+                        drivers_license,
+                        contact_relationship,
+                        phone_contact,
+                        phone_home,
+                        phone_biz,
+                        phone_cell,
+                        email,
+                        DOB,
+                        sex,
+                        race,
+                        ethnicity,
+                        status
```

Comment:
will need to get into PatientResource I assume

Reply:
Yes! That is correct. 

In the next PR I'm going to review the existing code to verify that we're supporting the "standard" [FHIR Search Parameters](https://github.com/openemr/openemr/pull/3385) for the fields in the list. I will also need to make sure that we've things aligned between the PatientService (DB) and the relevant FHIR Controller and Service classes.

Reply:
![image](https://user-images.githubusercontent.com/47683556/79848896-55e15d80-83df-11ea-993f-1f4593dfe0a3.png)

Will this be covered in your next PR or I can push a quick fix :)

Reply:
@yashrajbothra - feel free to go ahead and push your fix!